tests/MiniValidation.UnitTests/TypeDescriptorUtils.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.ComponentModel;
+
+namespace MiniValidation.UnitTests
+{
+    internal static class TypeDescriptorExtensions
```

Comment:
```suggestion
    internal static class TypeDescriptorHelpers
```

Or something other than `Extensions` as only want to use that for classes that contain actual extension methods. Or refactor these methods to be extension methods on `Type` or similar, e.g. `typeof(TestTypeForTypeDescriptor).GetProperty(nameof(TestTypeForTypeDescriptor.PropertyToBeRequired))?.AttachAttribute(_ => new RequiredAttribute())`

Reply:
You're totally right. I will take the extensions route.